### ISSUE TITLE
Fine-tune code for BiomedCLIP 

### ISSUE THREAD

**Issue by SwimmingLiu** [HealthX-Lab/MedCLIP-SAMv2#4]
Thank you for sharing this wonderful work!

I notice that the fine-tune code for the BiomedCLIP using our [DHN-NCE Loss](https://github.com/HealthX-Lab/MedCLIP-SAMv2/tree/main/loss) is not released. This fine-tune process is the significant part of your work, and it is very import for me to reproduce the whole process of your work. 

May you mind to share the fine-tune code for BiomedCLIP ?

Thank you once more and hope you doing well.

**Comment by TahaKoleilat** [comment]
Hi @SwimmingLiu,

Thank you for your interest in our work.

We unfortunately don't plan to release the fine-tuning code. You can directly follow the OpenCLIP tutorial [here](https://github.com/mlfoundations/open_clip/discussions/812) to fine-tune BiomedCLIP and you can change the loss function [here](https://github.com/mlfoundations/open_clip/blob/49eac2f27a5bb98a7f7ecc1154918880aa55256c/src/open_clip_train/train.py#L296) We also released our fine-tuned model [here](https://drive.google.com/file/d/1jjnZabUlc9_gpcP0d2nz_GNS-EGX0lq5/view?usp=sharing), so you can directly use it.

I will close this issue for now. If you have any other questions, please feel free to reopen.

**Comment by SwimmingLiu** [comment]
thank for your patient answer
